### ISSUE TITLE
do you need any help?

### ISSUE THREAD

**Issue by laysauchoa** [esiebert/rust-by-example-ptbr#1]

**Comment by esiebert** [comment]
Yeah, sure! I have translated all first 7 sections. I can leave section 8 for you. That means [this file](https://github.com/esiebert/rust-by-example-ptbr/blob/master/src/flow_control.md) and [all files under this folder](https://github.com/esiebert/rust-by-example-ptbr/tree/master/src/flow_control).

Currently the files are all empty and since **the file structure needs to be maintained**, I suggest copying the original ones (in english) and then translate them. You can find the original repo on [README.md](https://github.com/esiebert/rust-by-example-ptbr/blob/master/README.md).

When you are ready, please open a PR.
